config/config.js

```diff
@@ -0,0 +1,29 @@
+function getMergedDataFilename() {
+  const prefix = process.env.NODE_ENV === 'production' ? '' : 'dev-';
+  return `${prefix}merged-data.json`;
+}
+
+module.exports = {
+  app: {
+    name: 'sexual-health-service-data-combiner',
+  },
+  blobContainerName: process.env.AZURE_BLOB_CONTAINER_NAME || 'etl-output',
+  data: {
+    dir: {
+      current: './data/current',
+      latest: './data/latest',
```

Comment:
The `current` and `latest` folders do not exist causing an exception to be thrown during download.

Reply:
Mmh, I thought I'd tested that. I'll get it sorted.